src/portable/st/stm32_fsdev/dcd_stm32_fsdev.c

```diff
@@ -566,47 +585,77 @@ bool dcd_edpt_open (uint8_t rhport, tusb_desc_endpoint_t const * p_endpoint_desc
   (void)rhport;
   uint8_t const epnum = tu_edpt_number(p_endpoint_desc->bEndpointAddress);
   uint8_t const dir   = tu_edpt_dir(p_endpoint_desc->bEndpointAddress);
-
+  const uint16_t epMaxPktSize = p_endpoint_desc->wMaxPacketSize.size;
   // Isochronous not supported (yet), and some other driver assumptions.
+#ifndef NDEBUG
```

Comment:
please remove NDEBUG, I still prefer to check all parameters from application. We can use `CFG_TUSB_DEBUG` later on for this if user is desperate for flash. So far, there is no call for flash size just yet.